addons/escoria-core/game/core-scripts/esc/esc_room_manager.gd

```diff
@@ -85,12 +85,13 @@ func change_scene(room_path: String, enable_automatic_transitions: bool) -> void
 			ESCTransitionPlayer.TRANSITION_MODE.OUT
 		)
 
-		escoria.logger.debug(
-			"Awaiting transition %s (out) to be finished." % str(transition_id)
-		)
-
-		yield(escoria.main.scene_transition, "transition_done")
-
+		if transition_id != escoria.main.scene_transition.TRANSITION_ID_INSTANT:
+			escoria.logger.debug(
+				"Awaiting transition %s (out) to be finished." % str(transition_id)
```

Comment:
Is it redundant? `transition_id` holds an int, and I was under the impression that you'd need to use `%d` for such things; otherwise the interpolation would squawk. Is this not the case?

Reply:
"The %s seen in the example above is the simplest placeholder and works for most use cases: **it converts the value by the same method by which an implicit String conversion or str() would convert it**.  Strings remain unchanged, Booleans turn into either "True" or "False", an integral or real number becomes a decimal, other types usually return their data in a human-readable string." (https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_format_string.html)

Reply:
I swear I've had that crash out on me before, but then again maybe I'm thinking about Python. Anyway, you're right and I'll fix up.

Reply:
Fix for this pushed.

Reply:
Well I'm right until we find out it's not Python you were thinking of and it crashes out on you ;-)